ruma-api/tests/ui/02-invalid-path.rs

```diff
@@ -5,7 +5,7 @@ ruma_api! {
         description: "Does something.",
         method: GET,
         name: "newtype_body_endpoint",
-        path: "/_matrix/some/query/map/\u{ffff}",
+        path: "µ / ° / § / €",
```

Comment:
Haha, I actually meant including just one characters like that instead of `\u{ffff}`, but this works too ^^

Reply:
It's a really invalid path :laughing: